crud-framework-core/src/main/java/dev/krud/crudframework/crud/handler/CrudUpdateHandlerImpl.java

```diff
@@ -151,5 +151,64 @@ public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> Entity updat
 		return entity;
 	}
 
+    @Override
+    public <ID extends Serializable, Entity extends BaseCrudEntity<ID>> List<Entity> bulkUpdate(List<Entity> entities, HooksDTO<CRUDPreUpdateHook<ID, Entity>, CRUDOnUpdateHook<ID, Entity>, CRUDPostUpdateHook<ID, Entity>> hooks, boolean applyPolicies) {
+        if (entities.isEmpty()) {
+            throw new IllegalStateException("Cannot update an empty list of entities");
+        }
+
+        Class<Entity> entityClazz = (Class<Entity>) entities.get(0).getClass();
```

Comment:
See comment regarding EntityMetadataDTO

Reply:
I cant get the metadata without the entity class that i am getting from the list, its not reflection